### PR TITLE
order media so that latest added are top

### PR DESCRIPTION
**Pull request by fdb-hiroshima**
partial fix for #621

### CODE REVIEW

File: plume-models/src/medias.rs [hunk 1]
```diff
@@ -62,7 +62,14 @@ impl MediaCategory {
 impl Media {
     insert!(medias, NewMedia);
     get!(medias);
-    list_by!(medias, for_user, owner_id as i32);
+
+    pub fn for_user(conn: &Connection, owner: i32) -> Result<Vec<Media>> {
+        medias::table
+            .filter(medias::owner_id.eq(owner))
+            .order(medias::id.desc())
```
Comment (igalic) [inline plume-models/src/medias.rs]: is this guaranteed to be such?
do we have a create date anywhere?

Comment (fdb-hiroshima) [reply plume-models/src/medias.rs]: we don't have date, and the ids we use are monotonic counter, unless we overflow that 64b signed counter (not likely to happen)

Comment (igalic) [reply plume-models/src/medias.rs]: i wonder if anyone expects that editing a media pushes it to the top again…

Comment (AnaGelez) [reply plume-models/src/medias.rs]: We can't edit media information for the moment, so…

Comment (igalic) [reply plume-models/src/medias.rs]: well then… *pushes [resolve] button*

**Pull request merged**